cg/server/api.py

```diff
@@ -113,24 +97,25 @@ def families():
         records = db.cases_to_mip_analyze()
         count = len(records)
     else:
-        customer_obj = None if g.current_user.is_admin else g.current_user.customer
+        customer_objs = None if g.current_user.is_admin else g.current_user.customers
```

Comment:
```suggestion
        customer_objs: Optional[List[models.Customer]] = None if g.current_user.is_admin else g.current_user.customers
```

Reply:
I like the type hints since it makes one think if stuff can be done in other ways :)

Reply:
good point, fixed